docs/internals/release-process.txt

```diff
@@ -83,8 +126,8 @@ each version.
 * The current development master will get new features and bug fixes
   requiring major refactoring.
 
-* Patches applied to the master branch must also be applied to the last major
-  release, to be released as the next minor release, when they fix critical
+* Patches applied to the master branch must also be applied to the last feature
+  release, to be released as the next patch release, when they fix critical
```

Comment:
"must also be applied to the last feature release _branch_, to be released _in_ the next patch release _in that feature release series_" ?